misc/demo-c/analyze_raw_request.c

```diff
@@ -18,7 +18,7 @@
 #include <stdlib.h>
 #include <assert.h>
 #include "http_desync_guardian.h"
-#incldesync_ude "http_guardian_macros.h"
+#include "http_desync_guardian_macros.h"
```

Comment:
Thanks for pointing this out!
This line can be deleted. There used to be such a file, but it's no longer needed.

Reply:
I'll delete the line and modify the pull request, thank you!

```
$ git diff --cached
diff --git a/misc/demo-c/analyze_raw_request.c b/misc/demo-c/analyze_raw_request.c
index 20e90c5..894422e 100644
--- a/misc/demo-c/analyze_raw_request.c
+++ b/misc/demo-c/analyze_raw_request.c
@@ -18,7 +18,6 @@
 #include <stdlib.h>
 #include <assert.h>
 #include "http_desync_guardian.h"
-#include "http_desync_guardian_macros.h"
 #include "defs.h"
 
 #pragma clang diagnostic push
$ gcc -o analyze_raw_request analyze_raw_request.c ../../target/release/libhttp_desync_guardian.so
$ ls
README.md  analyze_parsed_request.c  analyze_raw_request  analyze_raw_request.c  defs.h  http_desync_guardian.h
$ ./analyze_raw_request 
Request is Ambiguous: "PUT", Ambiguous:  BothTeClPresent "Content-Length": " 10", tier: Bad
```


By the way, this outputs many errors.
```
$ gcc -o analyze_parsed_request analyze_parsed_request.c ../../target/release/libhttp_desync_guardian.so
```

Reply:
Oops. Just delete lines 144-153 (the `http_desync_guardian_initialize_metrics_settings` call and the comment block above).

Reply:
Now the both programs work well.
Thank you!

```
$ gcc -o analyze_parsed_request analyze_parsed_request.c ../../target/release/libhttp_desync_guardian.so
$ ls
README.md  analyze_parsed_request  analyze_parsed_request.c  analyze_raw_request  analyze_raw_request.c  defs.h  http_desync_guardian.h  http_desync_guardian.h~
$ ./analyze_parsed_request 
[1597856568] Method: "PUT", Type: Ambiguous, Count: 1
Non-compliant request:
Method="PUT", Url="_", Version="HTTP/1.1", Headers="Accept-Language": " _", tier: Compliant, "Cache-Control": " _", tier: Compliant, "Connection": " _", tier: Compliant, "Host": " _", tier: Compliant, "User-Agent": " _ _ _ _ _ _ _ _ _ _ _ _ _", tier: Compliant, "Transfer-Encoding": "chunked", tier: Compliant, "Content-Length": "1000", tier: Bad, "X-Empty-Header": "", tier: Compliant
Request is Ambiguous: "PUT", Ambiguous:  BothTeClPresent "Content-Length": "1000", tier: Bad
(!) Remove header "Content-Length": "1000"
```